site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -340,6 +341,28 @@ For high availability deployments with purchased support:
 * The FusionAuth engineering team will consult on product implementation and can offer "best practices" advice to further ensure system stability.
 * We size the instance properly for expected traffic, in consultation with the customer.
 
+== Load Testing
+
+include::docs/v1/tech/installation-guide/_load-testing-intro.adoc[]
+
+=== General Tips
+
+include::docs/v1/tech/installation-guide/_load-testing-tips.adoc[]
+
+=== Load Testing With FusionAuth Cloud
+
+Load testing is an important part of evaluating FusionAuth. However, it is important to load test in an environment that replicates your expected production environment. This means that if you are planning to use an HA cloud deployment for production, you should load test against that type of environment.
+
+When you are load testing against FusionAuth Cloud instances:
+
+* Please link:https://account.fusionauth.io/account/support/[open a support ticket] to let us know when you are planning to load test. We monitor CPU usage and other metrics and it can be helpful for us to know when you are planning to stress test your instance.
+* Store configuration as scripts which you can apply to your FusionAuth instance. This allows you to easily stand up and tear down different deployments to see what meets your needs.
+* If you have an edition with support, please link:https://account.fusionauth.io/account/support/[open a support ticket] if you'd like additional support or guidance on sizing or testing.
+
+Please don't use a FusionAuth Cloud Basic deployment to load test. The reason for this is that this tier is running on a single AWS EC2 instance. This single node is running FusionAuth, Elasticsearch and a PostgreSQL database. The node is therefore very resource constrained, and attempting to run load tests on this type of system is not recommended. The numbers you get from this type of test will not be valuable to you in context of planning your production deployment.
```

Comment:
Do we state anywhere else specifically that we use AWS EC2? 
If not, I would make this a bit more generic, such as a "single compute node".